metrics/web/test/auth/presentation/state/auth_notifier_test.dart

```diff
@@ -124,43 +228,219 @@ void main() {
       () {
         final userController = StreamController<User>();
 
-        when(receiveAuthUpdates.call())
-            .thenAnswer((realInvocation) => userController.stream);
+        when(receiveAuthUpdates()).thenAnswer((_) => userController.stream);
 
         authNotifier.subscribeToAuthenticationUpdates();
 
         expect(userController.hasListener, isTrue);
       },
     );
 
-    test(".isLoggedIn status is true after a user signs in", () async {
-      final user = User(id: 'id', email: email);
+    test(
+      ".subscribeToAuthenticationUpdates() delegates to receive user profile updates when the user is not null",
```

Comment:
It seems like this test is redundant because we have a test that checks that this method subscribes to the user profile updates.